javascript/apps/taiga/src/app/shared/comments/components/comment-detail/comment-detail.component.html

```diff
@@ -0,0 +1,99 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<ng-container *ngIf="model$ | async as vm">
+  <ng-container
+    *ngIf="
+      vm.project.userIsAdmin ||
+      (user.username === comment.createdBy.username && canComment)
+    ">
+    <ng-container *transloco="let t">
+      <tui-hosted-dropdown
+        [tuiDropdownAlign]="'right'"
+        [content]="showDeleteCommentConfirm ? confirmDelete : commentOptions"
+        [(open)]="commentOptionsState">
+        <button
+          role="menu"
+          appearance="action-button-2"
+          aria-haspopup="true"
```

Comment:
There is a weird animation when the 'modal' after the dropdown appears
![Peek 2023-07-10 12-25](https://github.com/taigaio/taiga/assets/5970959/145b5966-e0e3-4538-aa95-b1391b6d804a)